heyjaywilson/peakfit
# feat(exercises): :sparkles: Show a modal to select the exercises of an existing ExerciseList

## Description


In the View to see the details of an ExerciseList, we want to have a + button in the toolbar to add other exercises. 
To do that the SelectExercisesView is shown as a modal when the + button is tapped. 

### Related Issue






Closes #15 

## How Has This Been Tested?




Tested the next cases: 

Case 1 
* Create a new list 
* Navigate to the list
* Tap the + button in the toolbar 
* Select an exercise 
* Navigate back to the list 

**Result**: The exercise is shown in the list 

Case 2 
* Navigate to the list created in Case 1 
* Tap the + button in the toolbar
* Remove the exercise selected 
* Navigate back to the list 

**Result**: No exercises are shown 

 Case 3
* Navigate to the list created in Case 1 
* Tap the + button in the toolbar
* Select multiple exercises  
* Navigate back to the list 

**Result**: The correct exercises are shown 

Case 4: 
* Kill the app 
* Open the app again 
* Navigate to the list created in Case 1 

**Result**: The list created in Case 1 contains the right exercises  

## Screenshots (if appropriate):

![peakfit-screenrecording](https://github.com/user-attachments/assets/abd3d959-4f1c-4b14-8edc-325ed56a2519)


## Types of changes



- [ ] Developer tooling (fix or improve developer tooling)

## Checklist:
